### PR TITLE
Escape in quoted strings and fix the serializer tool

### PR DESCRIPTION
**Pull request by zbraniecki**
@Pike - r?

### CODE REVIEW

**Comment by Pike** [comment]
Is there a test for this?

**Comment by zbraniecki** [comment]
Added test and fixed the same in js serializer

File: l20n/format/serializer.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def dumpPattern(self, pattern):
                 str += self.dumpPlaceable(elem)
 
         if pattern['quoted']:
-            return u'"{}"'.format(str)
+            return u'"{}"'.format(str.replace('"', '\\"'))
```
Comment (stasm) [inline l20n/format/serializer.py:83]: @zbraniecki will this work for placeables which contain quotes required by the syntax, such as `{ "literal string" }` or `{ NUMBER($num, style="decimal" }`? I think we should only escape inside of `TextElements` which may be part of other expressions.  I remember that when I considered this bug I thought we'd need to use a flag passed into all `dumpPlaceable` methods which would pass it on to `dumpPattern` for nested patterns and escape in line 76.

**Comment by stasm** [comment]
`python-l20n` has been discontinued in favor of `python-fluent` which supports a newer version of the Fluent syntax and the AST.

**Pull request closed without merging**